### ISSUE TITLE
Shields damage

### ISSUE THREAD

**Issue by N00byEdge** [Blizzard/s2client-api#100]
Some attacks (like widow mines afaik) do extra damage vs shields. As far as I can tell, that is not a unit attribute in the api, and the damage bonus can therefore not be parsed from the weapon.

**Comment by KevinCalderone** [comment]
The widow mine attack is technically an auto-cast ability, so that is why it shows up as having no weapon data. In general we are currently not outputting damage information for abilities. That may be non-trivial to add as most abilities are custom scripted.

For units with regular weapons, the attribute bonuses should be in the UnitTypeData.

**Comment by N00byEdge** [comment]
Right, the issue is not that the damage is not provided for the ability but that when it is, `shields` is not a `sc2::Attribute `and can therefore not be a part of (any) damage bonuses.

**Comment by KevinCalderone** [comment]
Ah you are right. It looks like we are missing that one even for weapons.
